docs/ww3_doxy_tmpl.md

```diff
@@ -0,0 +1,68 @@
+# WAVEWATCH III Doxygen Header Reference
+<br>
+
+_Doxygen markup headers to be placed directly above the respective Fortran code units._
+
+### FILE
+```
+!> @file  <file name>
```

Comment:
Don't put the file name in the file.

Over time, as files get moved around, these will become incorrect. However, if never specified, doxygen will always use the correct file name.

So the top of our files look like this:

```
!>    @file
!>    @brief Pack up a Grid Definition Section (Section 3) and add it to
!>    a GRIB2 message.
!>    @author Stephen Gilbert @date 2000-05-01


```

Reply:
I was not aware doxygen pulled the name itself.  I'll make this change.  The PR just got accepted, so I'll discuss with Ali at a meeting we have later today to discuss how to add this in.  I'll resolve the conversation once it's done.

Reply:
The PR is not merged so you can still make changes. If it gets merged, you can just make a new PR.

Reply:
Oh I take that back, it has been merged, so just make a new PR to change the file with anything you'd like to be changed.

Reply:
Sounds good.  Will do.

Reply:
@edwardhartnett , please see new PR #593 .  This has been addressed.